src/scripts/main.js

```diff
@@ -1,7 +1,115 @@
 'use strict';
 
 // Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+import Game from '../modules/Game.class.js';
+
+const game = new Game();
 
 // Write your code here
+const gameScore = document.querySelector('.game-score');
+const startButton = document.querySelector('.buttom start');
```

Comment:
The selector '.buttom start' seems incorrect. It should likely be '.button.start' to correctly select the element with class 'button' and 'start'.